docs/source/sci_api_tutorial/PDBParser.rst

```diff
@@ -0,0 +1,174 @@
+==============================================
+PDBParser
+==============================================
+
+Briefs
+==============================================
+``PDBParser`` facilitates the conversion between PDB files and ``Structure`` objects. It primarily offers two functions: ``get_structure()`` for converting a PDB file into a ``Structure`` object, and ``save_structure()`` for performing the inverse operation.
+
+Input/Output
+==============================================
+
+.. panels::
+
+    :column: col-lg-12 col-md-12 col-sm-12 col-xs-12 p-2 text-left
+
+    .. image:: ../../figures/PDBParser_io.svg
+        :width: 100%
+        :alt: PDBParser_io
+
+PDBParser.get_structure()
+==============================================
+
+Input
+------------------------------------------------
+
+``path``
+    Specifies the file path of the PDB file.
+
+    .. admonition:: How to obtain
+
+        PDB files can be downloaded from the `Protein Data Bank <https://www.rcsb.org/>`_ or acquired from experimental results.
+
+Output
+------------------------------------------------
+
+``Structure()``
+    Returns the constructed ``Structure`` object.
```

Comment:
add a tip about we fix all the missing chains

Reply:
Done